src/main/java/com/gmail/goosius/siegewar/metadata/SiegeMetaDataController.java

```diff
@@ -49,7 +50,8 @@
 	private static LongDataField endTime = new LongDataField("siegewar_endTime", 0l);
 	private static LongDataField actualEndTime = new LongDataField("siegewar_actualEndTime", 0l);
 	private static StringDataField attackerSiegeContributors = new StringDataField("siegewar_attackerSiegeContributors", "");
-	
+	private static StringDataField primaryTownGovernments = new StringDataField("siegewar_primaryTownGovernments", "");

```

Comment:
From what I can gather this variable would be more aptly name primaryGovtUUID. 

There's code below to parse over a list of potential UUIDs, but I haven't been able to find any code where multiple UUIDs can be set. 

Is the potential Nation UUID that is recorded only ever the home-nation or can it be any nation that occupies the town as well?

Reply:
- This field is used to store which 'home nations' the town had during the course of the siege
- Example: 
  - The field could be: "[UUID-for-nation-A: 12] [UUID-for-town: 5] [UUID-for-nationB: 8] [UUID-for-nation-C: 3]"
  - The would reflect that the town change homed nations multiple times over the course of the siege, and at one point was nationless. (the numbers are total battle sessions)
- The occupying nation does not contribute to this list.

Reply:
Ok I have taken another look and the UUID is recorded via the BattleSession util. Can towns gain a nation during a siege?